python/ray/rllib/ppo/env.py

```diff
@@ -33,7 +35,8 @@ def step(self, actions, render=False):
                 observations.append(np.zeros(self.shape))
                 rewards.append(0.0)
                 continue
-            observation, reward, done, info = self.envs[i].step(action)
+            for j in range(self.extra_frameskip):
```

Comment:
we need to actually accumulate the reward from the frameskips here and sum it up into the reward that gets appended